### ISSUE TITLE
series download doesn't work

### ISSUE THREAD

**Issue by svalavuo** [aajanki/yle-dl#195]
Tried to download latest episodes of few series, but my script gives an error:
yle-dl --output-template '${title}-${episode}${date}' --resume --latestepisode --destdir /<destination directory>/sarjat/Murha\ paratiisissa https://areena.yle.fi/1-2229200

yle-dl 20190203: Download media files from Yle Areena and Elävä Arkisto
Copyright (C) 2009-2019 Antti Ajanki <antti.ajanki@iki.fi>, license: GPLv3

ERROR: Can't read https://areena.yle.fi/api/programs/v1/items.json?series=1-2229200&type=program&availability=ondemand&order=episode.hash%3Adesc%2Cpublication.starttime%3Adesc%2Ctitle.fi%3Aasc&app_id=89868a18&app_key=54bb4ea4d92854a2a45e98f961f0d7da&limit=100
Traceback (most recent call last):
  File "/root/.local/lib/python3.5/site-packages/yledl/http.py", line 86, in get
    r.raise_for_status()
  File "/root/.local/lib/python3.5/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://areena.yle.fi/api/programs/v1/items.json?series=1-2229200&type=program&availability=ondemand&order=episode.hash%3Adesc%2Cpublication.starttime%3Adesc%2Ctitle.fi%3Aasc&app_id=89868a18&app_key=54bb4ea4d92854a2a45e98f961f0d7da&limit=100
ERROR: Failed to parse a playlist
ERROR: No streams found

Just did a new git pull, but it still doesn't work. Am I doing something wrong?

**Comment by aajanki** [comment]
Thanks for reporting! This should now be fixed in the latest version in Github. It was caused by changes Yle has made in Areena.

**Comment by svalavuo** [comment]
Hi, Thanks! Now it works again!
Can you tell me why --latest doesn't work (with new version)?
I tried to use:
/yle-dl --output-template '${title}-${episode}${date}' --latestepisode --resume --destdir /<destination path>/Bancroft/ https://areena.yle.fi/1-4422251
For some reason it keeps downloading the first episode.
And should that --latest --resume -pair prevent downloading episode again? Or is there a better way?
And can those --output-template -variables be used in destdir as well?

**Comment by aajanki** [comment]
While fixing the series download, I made a mistake that caused --latestepisode to download the oldest episode intead of the newest. This is now fixed. I just released a new yle-dl version 20193131 with the fix: https://aajanki.github.io/yle-dl/index.html

--latest --resume is supposed to prevent re-loading but it might currently be broken because of the bug #183 

The output template variable can't unfortunately be used in destdir.
